Makefile

```diff
@@ -89,6 +89,13 @@ endif
 
 include scripts/standalone.mk
 
+#############################################################
+# Prints version message
+#############################################################
+.PHONY: version
+version:
+	@echo 2019.08
```

Comment:
I worry about hand-writing version numbers all over the place because we'll invariably forget one. Is there a way this can be automatically detected?